src/tasks/WasmAppBuilder/EmccCompile.cs

```diff
@@ -90,7 +92,7 @@ private bool ExecuteActual()
                     string depMetadata = srcItem.GetMetadata("Dependencies");
                     string[] depFiles = string.IsNullOrEmpty(depMetadata)
                                             ? Array.Empty<string>()
-                                            : depMetadata.Split(new char[] { ';' }, StringSplitOptions.RemoveEmptyEntries);
+                                            : depMetadata.Split(s_semicolon, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Ditto

Reply:
This project looks has .net framework build https://github.com/dotnet/runtime/blob/84b8408f1e12e68d663487d3fd482a78bf7bc942/src/tasks/WasmAppBuilder/WasmAppBuilder.csproj#L3